client/api/src/client.rs

```diff
@@ -279,6 +326,36 @@ pub struct BlockImportNotification<Block: BlockT> {
 	///
 	/// If `None`, there was no re-org while importing.
 	pub tree_route: Option<Arc<sp_blockchain::TreeRoute<Block>>>,
+	/// Handle to unpin the block this notification is for
+	_unpin_handle: UnpinHandle<Block>,
```

Comment:
AFAIR the `_` prefix prevents the "unused field" warning, but in this case it looks like it is used, so it shouldn't be necessary?

Reply:
> AFAIR the `_` prefix prevents the "unused field" warning, but in this case it looks like it is used, so it shouldn't be necessary?

Exactly. When I introduced the `UnpinHandle`, the compiler complained about the field never being read. But since I introduced the `into_unpin_handle` method this problem went away on its own.